nextflow.config

```diff
@@ -11,20 +11,21 @@ params {
 
     // Input options
     input                      = null
+
     // References
     genome                     = null
     igenomes_base              = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore            = false
-    // Qualimap options
-    bamqc_regions_file         = null
+    fasta_index                = null
+    bwameth_index              = null
```

Comment:
I don't think we should specify them here.
it's references, they should be defined by the getgenomeattributes and conf/igenomes.config

Reply:
its required as if fasta_index is not provided..pipeline creates it but, to start the pipeline these values need to be initialized